walkthroughs/1A-integrate-event-and-api-driven-apps/walkthrough.json

```diff
@@ -12,71 +12,134 @@
     "serviceInstances": [
       {
         "metadata": {
-          "name": "spring-boot-rest-http-crud"
+          "name": "postgresql-persistent"
         },
         "spec": {
-          "clusterServiceClassExternalName": "spring-boot-rest-http-crud",
-          "clusterServicePlanExternalName": "default"
+          "clusterServiceClassExternalName": "postgresql-persistent",
+          "clusterServicePlanExternalName": "default",
+          "parameters": {
+            "DATABASE_SERVICE_NAME": "orders-db",
+            "POSTGRESQL_DATABASE": "orders",
+            "POSTGRESQL_USER": "{{ user-username }}",
+            "POSTGRESQL_PASSWORD": "password"
+          }
         }
       },
       {
         "metadata": {
-          "name": "nodejs-messaging-work-queue-frontend"
+          "name": "rhmi-lab-nodejs-order-ui"
         },
         "spec": {
-          "clusterServiceClassExternalName": "nodejs-messaging-work-queue-frontend",
+          "clusterServiceClassExternalName": "rhmi-lab-nodejs-order-ui",
           "clusterServicePlanExternalName": "default",
           "parameters": {
             "MESSAGING_SERVICE_PASSWORD": "{{ enmasse-credentials-password }}",
             "MESSAGING_SERVICE_USER": "{{ enmasse-credentials-username }}",
             "MESSAGING_SERVICE_HOST": "{{ enmasse-broker-url }}"
           }
         }
+      },
+      {
+        "metadata": {
+          "name": "rhmi-lab-fuse-internal-orders-api"
+        },
+        "spec": {
+          "clusterServiceClassExternalName": "rhmi-lab-fuse-internal-orders-api",
+          "clusterServicePlanExternalName": "default",
+          "parameters": {
+            "PG_HOSTNAME": "orders-db.{{ walkthrough-namespace }}.svc",
+            "PG_DATABASE": "orders",
+            "PG_USERNAME": "{{ user-username }}",
+            "PG_PASSWORD": "password"
+          }
+        }
+      },
+      {
+        "metadata": {
+          "name": "rhmi-lab-order-management-ui"
+        },
+        "spec": {
+          "clusterServiceClassExternalName": "rhmi-lab-order-management-ui",
+          "clusterServicePlanExternalName": "default",
+          "parameters": {
+            "ORDERS_API_BASE_URL": "http://rhmi-lab-internal-fuse-api.{{ walkthrough-namespace }}.svc:8080"
+          }
+        }
       }
     ],
     "templates": [
       {
-        "apiVersion": "template.openshift.io/v1",
+        "apiVersion": "v1",
```

Comment:
@aidenkeating should this "just work" and install the template? I'm seeing errors about the `ClusterServiceClass does not exist` as shown in the screenshot. When I used this at RHTE I was manually loading the templates via scripts [like these](https://github.com/integr8ly/rhmi-solution-pattern-fuse-orders-api/tree/master/scripts) and that worked.

<img width="1440" alt="Screenshot 2019-11-08 at 12 16 05 PM" src="https://user-images.githubusercontent.com/1303687/68507881-c75a9480-0221-11ea-994a-da57d58537c5.png">

Reply:
Maybe I need to fix the `apiVersion` to reference `template.openshift.io`. Will dig a bit more.

Reply:
OK so that didn't do the trick.

I see that the installer references and does and `oc create` for them, so the installer needs to do it AFAICT. I'll PR there.